server/src/lfortran-accessors.ts

```diff
@@ -281,7 +287,21 @@ export class LFortranCLIAccessor implements LFortranAccessor {
     const fnid: string = "showDocumentSymbols(...)";
     const start: number = performance.now();
 
-    const flags = ["--show-document-symbols", "--continue-compilation"];
+    // run `echo "$(find . -type d | sed 's/^/-I/')"` and pass it as an argument to lfortran using spawnSync
+    // Execute the command within a shell
+    const find_output = spawnSync("echo $(find $(pwd) -type d | sed 's/^/-I/')", { shell: true });
```

Comment:
I think this has several problems:

* it will insert insane long command line arguments for large repos
* it will have collision problems for projects that have unrelated projects with modules with the same name
* it depends on `find` and `sed`, which is not available on all platforms (like Windows)